rosbridge_server/scripts/rosbridge_websocket.py

```diff
@@ -55,184 +60,240 @@
+
+        ##################################################
+        # Parameter handling                             #
+        ##################################################
+        retry_startup_delay = self.get_parameter('retry_startup_delay').value
+        retry_startup_delay = 2.0 if retry_startup_delay is None
+                                  else retry_startup_delay  # seconds
+
+
+        use_compression = self.get_parameter('use_compression').value
+        RosbridgeWebSocket.use_compression = False if use_compression is None
+                                                   else use_compression
+
+        # get RosbridgeProtocol parameters
+        fragment_timeout = self.get_parameter('fragment_timeout').value
+        RosbridgeWebSocket.fragment_timeout = RosbridgeWebSocket.fragment_timeout
+                                              if fragment_timeout is None
+                                              else fragment_timeout
+
+        delay_between_messages = self.get_parameter('delay_between_messages').value
+        RosbridgeWebSocket.delay_between_messages = RosbridgeWebSocket.delay_between_messages 
+                                                    if delay_between_messages is None
+                                                    else delay_between_messages
+
+        max_message_size = self.get_parameter('max_message_size').value
+        RosbridgeWebSocket.max_message_size = RosbridgeWebSocket.max_message_size
+                                              if max_message_size is None
+                                              else max_message_size
+
+        unregister_timeout = self.get_parameter('unregister_timeout').value
+        RosbridgeWebSocket.unregister_timeout = RosbridgeWebSocket.unregister_timeout
+                                                if unregister_timeout is None
+                                                else unregister_timeout
+
+        bson_only_mode = self.get_parameter('retry_startup_delay').value
+        bson_only_mode = False if bson_only_mode is None
+                               else bson_only_mode
+
+        if RosbridgeWebSocket.max_message_size == "None":
+            RosbridgeWebSocket.max_message_size = None
+
+        # SSL options
+        certfile = self.get_parameter('certfile').value
+        keyfile = self.get_parameter('keyfile').value
+        # if authentication should be used
+        authenticate = self.get_parameter('authenticate').value
+        RosbridgeWebSocket.authenticate = False if authenticate is None else authenticate
+
+        port = self.get_parameter('port').value
+        port = 9090 if port is None else port
+        
+        address = self.get_parameter('address').value
+        address = "" if address is None else address
+
+        # Publisher for number of connected clients
+        # QoS profile with transient local durability (latched topic in ROS 1).
+        client_count_qos_profile = qos_profile_default
```

Comment:
I think `qos_profile_default` is deprecated. Also, why not setting `depth` to 10? `queue_size` was 10 before.

Reply:
Thanks; addressed in https://github.com/RobotWebTools/rosbridge_suite/pull/419/commits/9161521da0b402b5fa6720e54cb61bd156ec58f6.